src/client/java/teammates/client/scripts/DataMigrationForFeedbackQuestionsToQuestions.java

```diff
@@ -97,11 +109,15 @@ protected void doOperation() {
                                     e);
                 } catch (EntityAlreadyExistsException e) {
                     // ignore if a copy of the old question already exists
-                    System.out.println("new qn type already exists for question:" + old.getIdentificationString());
+                    System.out.println("New question type entity already exists for question:" + old.getIdentificationString());
                 }
             }
+            
+            if (i % 100 == 0) {
+                System.out.println("Migrating the " + i + "'th Question");
```

Comment:
I am not sure about this actually. Printing for each question will create too much noise in the console logs when the script is run, and may make it hard to spot any error or warning.